action.yml

```diff
@@ -22,7 +26,7 @@ branding:
   color: "green"
 runs:
   using: "docker"
-  image: "docker://ghcr.io/trufflesecurity/trufflehog:latest"
+  image: "docker://ghcr.io/trufflesecurity/trufflehog:${{ inputs.version }}"
```

Comment:
I don't really think that the `image` field accepts the Github's expression syntax. It pretty much broke the workflow that I'm using. (Like what @james-callahan stated earlier)

One alternative is to give a default value not as seen on line 9, but to set it on this current line. In which I don't really know how to do that from the [expression documentation](https://docs.github.com/en/actions/learn-github-actions/expressions). So like in Javascript, you'd do:

```js
${{ input.version || 'latest' }}
```

But I guess that would be impossible because the expression syntax treats `||` as an `OR` operator.